nfcreader/src/main/java/company/tap/nfcreader/open/reader/TapNfcCardReader.java

```diff
@@ -68,12 +67,16 @@ public void disableDispatch() {
      */
     public boolean isSuitableIntent(Intent intent) {
         final Tag tag = intent.getParcelableExtra(NfcAdapter.EXTRA_TAG);
+        boolean isSuitable;
         if (tag == null) {
+            isSuitable =false;
             logger.debug("No TAG in intent");
             return false;
```

Comment:
Thank you 🌹
You just forget this `return false;` here, if the method is returned now the logger will never happen
and also remember to press `option + command + L` to format the code after you finish, because `isSuitable=true` missing spaces